### PR TITLE
update blog url in newsfeed

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

File: app/scripts/controllers/feed.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ angular.module('feed', ['stats'])
     };
   })
   .controller('NotificationsController', function ($scope, $sce, $http, $filter, FeedService, NotificationService, ClusterState, UdcSettings) {
-    var BLOG_URL = 'https://crate.io/blog/category/developernews';
+    var BLOG_URL = 'https://crate.io/blog';
```
Comment (chaudum) [inline app/scripts/controllers/feed.js]: `/blog/category/developernews` should be a http redirect to `/blog`!

**Comment by chaudum** [comment]
@mikethebeer check with @joemoe where to change the redirect. it's done in the nginx config in salt afaik

**Comment by mikethebeer** [comment]
@chaudum redirect changed on Fastly

**Comment by chaudum** [comment]
ack

**Pull request merged**